### PR TITLE
Virtualization: Maintenance: Add SLES15-SP3

### PR DESCRIPTION
**Pull request by pdostal**
Add SLES15SP3 to virtualization maintenance.

- Related ticket: [poo#91701](https://progress.opensuse.org/issues/91701)
- Needles: [gitlab.suse.de/openqa/os-autoinst-needles-sles!1534](https://gitlab.suse.de/openqa/os-autoinst-needles-sles/-/merge_requests/1534)
- Verification run: [XEN@15-SP3](http://openqa.qam.suse.cz/tests/21372) [KVM@15-SP3](http://openqa.qam.suse.cz/tests/21345)

### CODE REVIEW

File: lib/virt_autotest/common.pm [hunk 1]
```diff
@@ -153,6 +153,24 @@ if (get_var("REGRESSION", '') =~ /xen/) {
             distro       => 'SLE_15',
             location     => 'http://mirror.suse.cz/install/SLP/SLE-15-SP2-Full-GM/x86_64/DVD1/',
         },
+        sles15sp3PV => {
+            name         => 'sles15sp3PV',
+            autoyast     => 'autoyast_xen/sles15sp3PV_PRG.xml',
+            extra_params => '--os-variant sles12',
```
Comment (grisu48) [inline lib/virt_autotest/common.pm:159]: We probably should use `sle15sp3` and also update the other ones. I not possible at least reference a bsc number here why we need to use a old version.

Comment (pdostal) [reply lib/virt_autotest/common.pm:159]: OK, this is actually bigger issue:

The problem is that on older hosts we don't have those newer options.
I suggest to create a ticket and discuss / plan / track the issue separately.

Comment (alice-suse) [reply lib/virt_autotest/common.pm:159]: For very old hosts that are LTSS for a long time already, maybe such workaround can be used. But for newer products, eg sle12sp4/5, sle15spx, the correct os-variant should be there, right? If not, some bug should be opened?

**Review by alice-suse — COMMENTED** [review]
Thanks for the quick code support for sle15sp3, Pavel! Similar comment with Felix. BTW, will wait for the validation run before merging it.

File: lib/virt_autotest/common.pm (hunk 1 above)
Comment (grisu48) [reply lib/virt_autotest/common.pm:159]: I've created https://progress.opensuse.org/issues/92782 for it, so I guess we can leave this as it is and find a solution elsewhere.

Comment (pdostal) [reply lib/virt_autotest/common.pm:159]: I'm marking this as resolved here as we have separate ticket for this.

**Comment by alice-suse** [comment]
@pdostal Do you plan to add verification jobs? I am think about when to merge it.

**Review by alice-suse — APPROVED** [review]

**Comment by alice-suse** [comment]
Good job! Thanks for the support!

**Pull request merged**